<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>750</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>1011</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_Tools">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>650</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>151</width>
          <height>631</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>240</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255)</string>
           </property>
           <property name="title">
            <string>  Nástroje pro přehrávání</string>
           </property>
           <widget class="QPushButton" name="pushButton_Play">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>40</y>
              <width>45</width>
              <height>60</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recources.qrc">
              <normaloff>:/play.png</normaloff>:/play.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>60</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_Stop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>45</width>
              <height>60</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border : none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recources.qrc">
              <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>60</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_Pause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>110</y>
              <width>40</width>
              <height>60</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recources.qrc">
              <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>60</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_Forward">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>110</y>
              <width>60</width>
              <height>61</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recources.qrc">
              <normaloff>:/forward.png</normaloff>:/forward.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_Rewind">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>1</x>
              <y>114</y>
              <width>52</width>
              <height>51</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recources.qrc">
              <normaloff>:/rewind.png</normaloff>:/rewind.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>54</width>
              <height>57</height>
             </size>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox_Delay">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>180</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>32000</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>4</x>
              <y>180</y>
              <width>91</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>čas mezi tahy [ms]</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox_MovesCount">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>210</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>213</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>skočit o počet tahů.</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
           </property>
           <property name="title">
            <string>        Nástroje pro hru</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255)</string>
           </property>
           <property name="title">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="portLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>630</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>server running on port: </string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_Game">
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>650</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget_Games">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>651</width>
          <height>650</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>650</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>651</width>
          <height>630</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_Moves">
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_Moves">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>211</width>
          <height>650</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>650</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>650</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
        </property>
        <property name="title">
         <string>Moves</string>
        </property>
        <widget class="QTextEdit" name="lineEdit_Moves">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>191</width>
           <height>631</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border: none;
padding-left: 20px;</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="overwriteMode">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_OpenReplay"/>
    <addaction name="action_SaveIcp"/>
    <addaction name="action_SaveXml"/>
    <addaction name="action_Help"/>
   </widget>
   <widget class="QMenu" name="menuHra">
    <property name="title">
     <string>Hra</string>
    </property>
    <addaction name="action_LocalLabel"/>
    <addaction name="action_NewLocalGame"/>
    <addaction name="action_NewLocalGameVs"/>
    <addaction name="separator"/>
    <addaction name="action_NetLabel"/>
    <addaction name="action_NewNetGame"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menuHra"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="myStatusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>Otevřít</string>
   </property>
  </action>
  <action name="action_SaveIcp">
   <property name="text">
    <string>Uložit ICP syntaxi</string>
   </property>
  </action>
  <action name="action_SaveXml">
   <property name="text">
    <string>Uložit XML</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Zavřit</string>
   </property>
  </action>
  <action name="action_LocalLabel">
   <property name="text">
    <string>Lokální</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="action_NetLabel">
   <property name="text">
    <string>Síťová</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="action_NewLocalGame">
   <property name="text">
    <string>Nová hra proti CPU</string>
   </property>
  </action>
  <action name="action_NewNetGame">
   <property name="text">
    <string>Založit hru proti hráči</string>
   </property>
  </action>
  <action name="action_DisconnectGame">
   <property name="text">
    <string>Odpojit aktuální hru</string>
   </property>
  </action>
  <action name="action_NewLocalGameVs">
   <property name="text">
    <string>Nová hra proti hráči</string>
   </property>
  </action>
  <action name="action_OpenReplay">
   <property name="text">
    <string>Otevřít replay</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>Nápověda</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
